bin/yaml/ygen.yaml

```diff
@@ -0,0 +1,11 @@
+compilers:
+  ygen:
+    ylc:
+      type: ziparchive
+      fetch: https://github.com/Cr0a3/ygen/tree/build
```

Comment:
This is missing the `url:` parameter. I suspect you need that. Maybe `fetch` should be `url` ?

```
RuntimeError: Missing required key 'url' in compilers/ygen/ylc newest
```

If I make that change locally then it does'nt know how to find "newest". You URL needs to somehow contain `{name}` in order to specify the target.

I can't see any releases on https://github.com/Cr0a3/ygen/releases so I don't know what URL you need to specify to download the built compiler.

Reply:
I made a little script which automatically builds my project and force pushes the changes to the build branch which this script should download

Reply:
It does not do that.